config/default-config.yml

```diff
@@ -191,6 +191,13 @@ network-config:
   gossipsub-rpc-metrics-inspector-workers: 1
   # The size of the queue used by worker pool for the control message metrics inspector
   gossipsub-rpc-metrics-inspector-cache-size: 100
+  # Defines the duration of time, after the node startup,
+  # during which the scoring registry remains inactive before penalizing nodes.
+  # Throughout this startup silence period, the application-specific penalty
+  # returned for all nodes will be 0, and any invalid control message notifications
+  # will be ignored. This configuration allows nodes to stabilize and initialize before
+  # applying penalties or processing invalid control message notifications.
+  gossipsub-scoring-registry-startup-silence-duration: 20m
```

Comment:
how did you come up with this default, and the min `10m`? This seems too long for some node types and potentially too short for others (ENs). rather than a timer, is it possible to trigger a signal when startup is complete?

Reply:
@kc1116 maybe we can use this `node.Ready` channel instead of a silence period.

https://github.com/onflow/flow-go/blob/cd7acd64b56a35fbfa13af84b908f4e7d451f61e/cmd/node.go#L98

Reply:
https://github.com/onflow/flow-go/pull/5084/commits/b8a2fd234db0be52b9722703b512c77744622bfc The silence period default duration has been changed to 1 hour. [Here](https://github.com/dapperlabs/flow-internal/issues/1906) is the follow up issue to create a component that can notify other components when it is ready